1 table.sql

```diff
@@ -0,0 +1,55 @@
+use ScenicHotelDB
+go
+drop table if exists guest 
+go 
+create table dbo.guest(
+    GuestId int not null identity primary key,
+    FirstName varchar(25) not null 
+        constraint ck_Guest_first_name_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(25) not null 
+        constraint ck_Guest_last_name_cannot_be_blank check(LastName <> ''),
+    AmountOfPeople int not null 
+        constraint ck_Guest_amount_of_people_must_be_greater_than_zero check(AmountOfPeople > 0),
+    DateOfBirth date not null,
+    CreditCardNumber char(19) not null 
+        constraint ck_Guest_credit_card_number_must_be_16_digits_separated_by_a_space_after_each_4_digits
+            check(CreditCardNumber like '[0-9][0-9][0-9][0-9] [0-9][0-9][0-9][0-9] [0-9][0-9][0-9][0-9] [0-9][0-9][0-9][0-9]'),
+    ExpirationDate char(5) not null 
+        constraint ck_Guest_expiration_date_must_be_in_the_format_of_the_month_and_year_separated_by_slash check(ExpirationDate like '[0-1][0-9]/[0-9][0-9]'),
+    RoomType varchar(15) not null 
+        constraint ck_Guest_room_type_must_be_one_of_the_4_possible_options 
+            check(RoomType in('Suite', 'Standard Room', 'Connecting Room', 'Senior Room')),
+    RoomNumber int not null
+        constraint ck_Guest_room_number_must_be_between_100_and_399 check(RoomNumber between 100 and 399),
+    DateCheckedIn date not null
+        constraint ck_Guest_date_checked_in_must_be_after_oppening_date_and_cannot_be_future check(DateCheckedIn between '1998-01-01' and getdate()),
+    DateCheckedOut date null 
+        constraint d_Guest_date_checked_out default null
```

Comment:
I am almost sure that there is no need to specify a default for a null its a default anyway.

Reply:
I removed it